extendedcopy.go

```diff
@@ -45,7 +58,16 @@ func ExtendedCopy(ctx context.Context, src GraphTarget, srcRef string, dst Targe
 		return ocispec.Descriptor{}, err
 	}
 
-	if err := ExtendedCopyGraph(ctx, src, dst, node); err != nil {
+	// TODO: manifest filter? 1. media type filter 2. platform filter
```

Comment:
The platform type filtering depends on the client, oras-go only need to provide the manifestFilter options.

For the mediatype, manifestFilter can do the manifest filtering but we still need downEdgeFilter support.